types/amplitude-js/index.d.ts

```diff
@@ -127,6 +129,21 @@ export class AmplitudeClient {
     Revenue: typeof Revenue;
 }
 
+export interface CookieStorageOptions {
+    expirationDays?: number | undefined;
+    domain?: string | undefined;
+    secure?: boolean | undefined;
+    sameSite?: 'Lax' | 'Strict' | 'None' | undefined;
```

Comment:
should this be in `BaseCookieOptions` and `CookieStorageOptions` should extend it? (no need for BaseCookie)

Reply:
I'm sorry, I don't think I understand. Do you mean to reuse the typing from `Config.sameSiteCookie`? We don't have any `BaseCookie` typings ...